### ISSUE TITLE
Weapon lasers and lights do not seem to stay in line with viewmodel

### ISSUE THREAD

**Issue by tiltedcan** [yohjimane/xray-16#29]
**Describe the bug**
during any rapid turn weapon lasers and light will lead where they seem they should be by quite a bit

**To Reproduce**
1. Boot game and acquire weapon capable of tactical light and laser upgrade modules
2. equip laser and light modules
3. activate either the laser or the light
4. Turn rapidly or turn back and forth rapidly
5. See error- laser and light emissions do not appear to follow viewmodel

**Expected behavior**
laser and light emissions will stay in line with the viewmodel so that the player can use them effectively

**Screenshots**

https://github.com/yohjimane/xray-16/assets/12876132/d19a9dcf-286c-4f6a-a670-2fd8e2e5ec89

https://github.com/yohjimane/xray-16/assets/12876132/e2be0b64-3e9f-44a7-8bf4-d219e805e411

**BugTrap error report**
N/A

**Desktop**
 - OS: Windows 11 (AtlasOS)
 - OpenXRay Gunslinger  v0.0.1 with hotfix  v0.0.9

**Comment by tiltedcan** [comment]
Something that isn't immediately obvious from the footage I've posted here is how much these emissions jitter when moving as well, are the emitters attached to the view model in a different way than original gunslinger?

**Comment by tiltedcan** [comment]
did this get updated with hotfix v0.1.02? it seems to be way better 
edit: the lasers got way better, flashlight still shows same issue

**Comment by yohjimane** [comment]
> did this get updated with hotfix v0.1.02? it seems to be way better edit: the lasers got way better, flashlight still shows same issue

I committed this on accident i think, I'm still looking for a solid fix unfortunately. The solution present in the last build makes the laser direction a bit inaccurate (you can notice it while sprinting and some other actions)

**Comment by tiltedcan** [comment]
Yeah I did notice that behavior, editorially, I think it's an improvement, it makes the lasers much more usable. So it might be an unfinished solution but I think it's definitely on the right track.
